.changeset/flat-snails-admire.md

```diff
@@ -0,0 +1,11 @@
+---
+"@spectrum-css/infieldprogresscircle": major
+---
+
+## Infield Progresscircle S2 Migration
+
+In-field progress circle is a new component created to replace progress circle (size S) in t-shirt size components such as: buttons, combo boxes, pickers. Comes in four sizes: (S, M, L, XL), has updated color variants (default, white, black), and unified track thickness.
```

Comment:
Maybe we can make it clearer that we updated button, picker, and textfield to use infield progress circle? 

```suggestion
In-field progress circle is a new component created to replace progress circle (size S) in t-shirt size components. The button, textfield, combo box, and picker `template.js` files have all been updated to call for infield progress circles. This component comes in four sizes: (S, M, L, XL), has updated color variants (default, white, black), and has a unified track thickness.
```